libnymea/interfaces/irrigation.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "power",
+    "description": "The irrigation interface is used for irrigation systems. It is used for water pumps that power any sort of irrigation. The power state turns irrigation on and off."
```

Comment:
it also works for and with valves, does not necessarily need a pump.

Reply:
fixed